docs/lib/graphqlapi/fragments/android/getting-started.md

```diff
@@ -1,41 +1,17 @@
-You can integrate with Amplify framework using the following steps:
+> ### Prerequisites
+> * An Android project targeting Android API level 16 (Android 4.1) or above.
```

Comment:
Can we be consistent with punctuation on bullets? I'd prefer without, but "both" is the wrong answer. Also! I wonder if we can link to Google's Android documentation here directly to their getting started steps.

Reply:
Same. prefer without punctuation on bullets

Reply:
Good suggestions!  I will:
 - remove punctuation
 - add link to Google Android docs